### PR TITLE
feat: update use cases diagrams

### PR DESCRIPTION
**Pull request by nikvoblikov**
**This PR:**
- Updates to diagrams for use cases;

**Preview**
- Preview [Use cases page](https://deploy-preview-397--cilium.netlify.app)

**Steps to test**
- [x] Open the preview pages;
- [ ] Confirm that everything looks and works as expected;

### CODE REVIEW

**Review by prosazhin — APPROVED** [review]

File: src/images/pages/usecase/network-4.png
Comment (xmulligan) [inline src/images/pages/usecase/network-4.png:1]: Can you center "CiliumClusterwideNetworkPolicy"?

File: src/images/pages/usecase/network-2.png
Comment (xmulligan) [inline src/images/pages/usecase/network-2.png:1]: this this an actual screenshot of hubble?

File: src/images/pages/usecase/cni-2.png
Comment (xmulligan) [inline src/images/pages/usecase/cni-2.png:1]: is this a screenshot from grafan or something you created?

File: src/images/pages/usecase/cni-1.png
Comment (xmulligan) [inline src/images/pages/usecase/cni-1.png:1]: missing iptables overhead

File: src/images/pages/usecase/clustermesh-1.png
Comment (xmulligan) [inline src/images/pages/usecase/clustermesh-1.png:1]: missing the zombies

File: src/images/pages/usecase/bandwidth-1.png
Comment (xmulligan) [inline src/images/pages/usecase/bandwidth-1.png:1]: can you move the upper stack box out of line as it was before?

**Comment by xmulligan** [comment]
@prosazhin are we still waiting on something here? Or can you address the comments above?

**Comment by prosazhin** [comment]
@xmulligan Hi. We made changes to the comments. Please take a look.

**Comment by xmulligan** [comment]
@prosazhin They all look good now. Are you ready to merge?

**Comment by prosazhin** [comment]
@xmulligan We need more time for internal review. I will let you know when we can merge.

**Review by americano98 — APPROVED** [review]

**Comment by prosazhin** [comment]
@xmulligan Done! You can merge now

**Review by xmulligan — APPROVED** [review]

**Comment by xmulligan** [comment]
Great, thanks!

**Pull request merged**